xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -49,10 +49,19 @@
         <Parameter Name="source" Type="System.ReadOnlySpan&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">The read-only span to read.</param>
+        <summary>Reads an <see cref="T:System.Int16" /> out of a read-only span of bytes as big endian.</summary>
+        <returns>The big endian value.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Reads exactly 2 bytes from the beginning of the span.
+
+          ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="source" /> is smaller than <see cref="T:System.Int16" />.</exception>
```

Comment:
Imo, "source smaller than int16" would be confusing since we are talking about span.Length and the size of int16.

```suggestion
        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="source" /> is too small to contain an <see cref="T:System.Int16" />.
```